Assets/Scripts/PlayerController.cs

```diff
@@ -1,38 +1,54 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class PlayerController : MonoBehaviour
 {
-    public float rotationSpeed = 360;
-    public float acceleration = 100;
+    public Rigidbody2D playerRb;
+    public Camera camera;
```

Comment:
My Unity editor cries, because there is a depracated property in the base class that is called `camera`.
We should rename this variable to something else (ie. cam, mainCamera, worldCamera, etc..).

Also we don't really need to assign this from the editor. (makes life a bit more difficult if we want to make another map for example). We can assign the variable in the *Start* function with `Camera.main`